### PR TITLE
updating certain tests to be ITs

### PR DESCRIPTION
**Pull request by rcordeau**
This change is [Reviewable](https://reviewable.io/reviews/smarterapp/rdw_ingest/122)

### CODE REVIEW

**Comment by rcordeau** [comment]
Thank you for approving!

File: exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultTDSReportProcessorTest.java [hunk 1]
```diff
@@ -17,23 +15,12 @@
...
-import org.springframework.test.context.junit4.SpringRunner;
 
 import java.text.ParseException;
 
 import static org.mockito.Matchers.any;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```
Comment (malaffoon) [inline exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultTDSReportProcessorTest.java]: bad splat, bad!

Comment (rcordeau) [inline exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultTDSReportProcessorTest.java]: Thank Intellij for that.

Comment (rcordeau) [inline exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultTDSReportProcessorTest.java]: I ran the "Optimize Imports" on that file.

Comment (malaffoon) [inline exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/service/impl/DefaultTDSReportProcessorTest.java]: Did you apply the RDW Code Scheme (https://confluence.fairwaytech.com/display/SBACDW/Development+%28MacOS%29+Setup)?

**Pull request merged**